### PR TITLE
Added numbro@1.1.1

### PR DESCRIPTION
**Pull request by BenjaminVanRyseghem**

### CODE REVIEW

File: ajax/libs/numbro/package.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  },
+  "scripts": {
+    "test": "grunt test"
+  },
+  "npmName": "numbro",
+  "npmFileMap": [
+    {
+      "basePath": "/dist",
+      "files": [
+        "*"
```
Comment (PeterDaveHello) [inline ajax/libs/numbro/package.json]: I think it should be `**`

Comment (BenjaminVanRyseghem) [inline ajax/libs/numbro/package.json]: Ask @Graham42 :smile:

Comment (PeterDaveHello) [inline ajax/libs/numbro/package.json]: That's no need to do that, this config is only for cdnjs, not in nubro's repo.

Comment (BenjaminVanRyseghem) [inline ajax/libs/numbro/package.json]: Then I will say I used `cdnjs-importer` as explained in the doc, so I have no clue :smile

Comment (PeterDaveHello) [inline ajax/libs/numbro/package.json]: You still need to fix that.

Comment (BenjaminVanRyseghem) [inline ajax/libs/numbro/package.json]: Could you please explain why?

Comment (PeterDaveHello) [inline ajax/libs/numbro/package.json]: There is another directory in it.

**Comment by PeterDaveHello** [comment]
Would you please squash them into only one commit as readme file said? Thanks.

**Comment by BenjaminVanRyseghem** [comment]
I try, but with a 10Go repo, git is dying :cry: 

When I clone the repo, it's dirty, I tried `git reset --hard HEAD` or `git checkout -- ajax/libs/sortable/package.json`

Because of that, I can't rebase

**Comment by PeterDaveHello** [comment]
Are you using Mac OS? ref: #3650

**Comment by BenjaminVanRyseghem** [comment]
Indeed I am :/

**Comment by PeterDaveHello** [comment]
Okay, I'll merge it this time.

**Comment by BenjaminVanRyseghem** [comment]
Thanks :wink:

**Comment by PeterDaveHello** [comment]
np

**Comment by Graham42** [comment]
Numbro is showing up if I do a search using the API http://api.cdnjs.com/libraries?search=numbro but not when searching on the website https://cdnjs.com/. Should I open an issue for this? Or is there some wait time for the site to get updated?

**Comment by PeterDaveHello** [comment]
I think it works well now.
In API search, it'll only show the mainfile, AKA filename field in package.json.

**Comment by Graham42** [comment]
Thanks

**Pull request merged**